windows/RNSVG/RadialGradientView.cpp

```diff
@@ -53,40 +52,52 @@ void RadialGradientView::Unload() {
 }
 
 void RadialGradientView::CreateBrush() {
-  auto const &canvas{SvgRoot().Canvas()};
-  auto const &resourceCreator{canvas.try_as<ICanvasResourceCreator>()}; 
-  Brushes::CanvasRadialGradientBrush brush{resourceCreator, m_stops};
+  auto const root{SvgRoot()};
 
-  SetPoints(brush, {0, 0, canvas.Size().Width, canvas.Size().Height});
+  com_ptr<ID2D1DeviceContext> deviceContext{get_self<D2DDeviceContext>(root.DeviceContext())->Get()};
+
+  winrt::com_ptr<ID2D1GradientStopCollection> stopCollection;
+  winrt::check_hresult(deviceContext->CreateGradientStopCollection(&m_stops[0], static_cast<uint32_t>(m_stops.size()), stopCollection.put()));
+
+  D2D1_RADIAL_GRADIENT_BRUSH_PROPERTIES brushProperties;
```

Comment:
Is this struct being passed in completely uninitialized?

Reply:
those values are set in SetPoints below, but yes. I've now changed it to initialize with default values.